server/base/src/main/java/org/apache/accumulo/server/security/handler/PermissionHandler.java

```diff
@@ -138,11 +138,6 @@ void cleanNamespacePermissions(Namespace.ID namespace)
    */
   void initUser(String user) throws AccumuloSecurityException;
 
-  /**
-   * Initializes a new user
-   */
-  void initTable(String table);
```

Comment:
This may have been kept as pseudo-API (it's a pluggable component).

Reply:
Most likely but it was never called anywhere.  Perhaps NewTableConfig made this obsolete.  And if thats the case, we could deprecate and add a comment.

Reply:
This was originally supposed to be called when a new table was created... to initialize the permissions set in whatever underlying implemented storage. In our implementations, I think it wasn't needed, so it should have been a NOOP, but it should still have been called (I think).

Reply:
I reverted and marked this method deprecated.  This will at least inform anyone overriding the method that it currently isn't used.